src/components/post/styles.js

```diff
@@ -143,6 +143,13 @@ export const Content = styled.div`
       border-bottom: 1px solid #808080;
     }
   }
+  
+  li {
```

Comment:
Tomuhle nerozumím – v #73 jsem chtěl říct, že tam máme menší `line-height` než jinde v textu, což ale nesouvisí s marginem první položky. Su mimo? :)

Reply:
Moje zbrklost, hned opravím.

Reply:
S těmi procenty nejsem úplně spokojen, ale na to teď nemám úplně prostor. (Jsou i u odstavce)